lib/tokenizer/tokenize.js

```diff
@@ -154,10 +154,10 @@ function intoTokens(source, externalContext, internalContext, isNested) {
       // comment start within block preceded by some content, e.g. div/*<--
       metadatas.push(metadata);
       buffer.push(character);
-      buffers.push(buffer.slice(0, buffer.length - 2));
+      buffers.push(buffer.slice(0, -2));
```

Comment:
BTW `0` can also be skipped here like in the next `slice` call, since it's the default. We can go either way, but I'd suggest that we are consistent